collector/systemd_linux.go

```diff
@@ -188,6 +183,13 @@ func (c *systemdCollector) Update(ch chan<- prometheus.Metric) error {
 	}
 	defer conn.Close()
 
+	systemdVersion := c.getSystemdVersion(conn)
+	if systemdVersion < minSystemdVersionSystemState {
+		level.Debug(c.logger).Log("msg", "Detected systemd version is lower than minimum, some systemd state and timer metrics will not be available", "current", systemdVersion, "minimum", minSystemdVersionSystemState)
+	}
+	ch <- prometheus.MustNewConstMetric(
+		c.systemdVersionDesc, prometheus.GaugeValue, systemdVersion)
```

Comment:
Trailing `)` should be on a newline. You should `gofmt` this file

Reply:
Yea, this was cut-n-paste from the original lines.